cmd/cloud/installation_flag.go

```diff
@@ -89,10 +110,34 @@ func (flags *installationPatchRequestOptions) addFlags(command *cobra.Command) {
 	command.Flags().StringVar(&flags.image, "image", "mattermost/mattermost-enterprise-edition", "The Mattermost container image to use.")
 	command.Flags().StringVar(&flags.size, "size", model.InstallationDefaultSize, "The size of the installation. Accepts 100users, 1000users, 5000users, 10000users, 25000users, miniSingleton, or miniHA. Defaults to 100users.")
 	command.Flags().StringVar(&flags.license, "license", "", "The Mattermost License to use in the server.")
-
 	command.Flags().StringArrayVar(&flags.mattermostEnv, "mattermost-env", []string{}, "Env vars to add to the Mattermost App. Accepts format: KEY_NAME=VALUE. Use the flag multiple times to set multiple env vars.")
 	command.Flags().BoolVar(&flags.mattermostEnvClear, "mattermost-env-clear", false, "Clears all env var data.")
+}
+
+func (flags *installationPatchRequestOptions) GetPatchInstallationRequest() *model.PatchInstallationRequest {
+	request := model.PatchInstallationRequest{}
```

Comment:
Don't we want to use `default` flag values? In this case, If **CLI** doesn't set a flag explicitly, the pointer will be `nil` which doesn't match with previous behavior.

Reply:
That's the purpose of this PR. Since this is a patch operation, we do not want to use all the default values since we may only want to update one attribute.

Reply:
Thats right. Thanks